data/secscan_model/secscan_v4_model.py

```diff
@@ -149,9 +169,28 @@ def needs_reindexing_query(indexer_hash):
         )
 
         for candidate, abt, num_remaining in iterator:
-            # TODO(alecmerdler): Call out to Clair v4 to `/index` the manifest
-            logger.info("Clair v4 client not yet implemented, passing for now")
-            pass
+            manifest = ManifestDataType.for_manifest(candidate, None)
+            layers = registry_model.list_manifest_layers(manifest, self.storage, True)
+
+            logger.info(
+                "Indexing %s/%s@%s"
+                % (candidate.repository.namespace_user, candidate.repository.name, manifest.digest)
+            )
+
+            (report, state) = self._secscan_api.index(manifest, layers)
+
+            mss = ManifestSecurityStatus(
+                manifest=candidate,
+                repository=candidate.repository,
+                error_json=report["err"],
+                index_status=IndexStatus.FAILED
```

Comment:
Indentation is odd here... did you run `black`? If so, move this conditional into its own var